schema/app/DoctrineMigrations/Version20191226104551.php

```diff
@@ -0,0 +1,764 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20191226104551 extends AbstractMigration
```

Comment:
We should start using Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration for migrations